app/queues/dns/dns-flow.server.ts

```diff
@@ -1,23 +1,26 @@
 import { FlowProducer } from 'bullmq';
 import {
-  addDbRecordQueueName,
-  createDnsRecordQueueName,
+  dbRecordUpdateQueueName,
+  dnsRecordUpdateQueueName,
   checkDnsStatusQueueName,
   syncDbStatusQueueName,
-} from './create-record-worker.server';
+} from './dns-record-worker.server';
 import { redis } from '~/lib/redis.server';
+import { WorkType } from './dns-record-worker.server';
 
 import type { FlowJob } from 'bullmq';
-import type { JobRecord } from './create-record-worker.server';
+import type { JobRecord } from './dns-record-worker.server';
 
 const flowProducer = new FlowProducer({ connection: redis });
 
 export const addDnsRequest = async ({ username, type, name, value }: JobRecord) => {
+  const fullRecordName = `${name}.${username}.${process.env.ROOT_DOMAIN}`;
```

Comment:
Similar to what I told @dadolhay, we should probably have a common function in `app/utils.ts` or something for doing this kind of thing.  I had to write something similar in https://github.com/DevelopingSpace/starchart/blob/main/app/models/user.server.ts#L27.  One reason to centralize this is that we have to deal a `.` in a `username`.

Maybe we could do something like this:

```ts
// app/utils.ts

/**
 * Remove invalid/unwanted characters from a username. In the case
 * of faculty/admins, we will have a `.` in the username, which we
 * don't want to use as part of domain names.
 * @param username The user's username (e.g., `jsmith` or `john.smith`)
 */
function cleanUsername(username: PrismaUser['username']) {
  return username.replace(/\./g, '');
}

/**
 * Create the domain for a user, using their username
 * @param username The user's username (e.g., `jsmith` or `john.smith`)
 * @returns string jsmith -> jsmith.starchart.com
 */
function buildUserBaseDomain(username: PrismaUser['username']) {
  return `${cleanUsername(username)}.${process.env.ROOT_DOMAIN}`;
}

export function buildDomain(username: PrismaUser['username'], name?: string) {
  if(name) {
    return `${name}.${buildUserBaseDomain(username)}`
  }
  return buildUserBaseDomain(username);
}
```